base/logging_config.go

```diff
@@ -26,17 +25,23 @@ type LoggingConfig struct {
 	DeprecatedDefaultLog *LogAppenderConfig `json:"default,omitempty"` // Deprecated "default" logging option.
 }
 
-func (c *LoggingConfig) Init() error {
+func (c *LoggingConfig) Init(defaultLogFilePath string) (err error) {
 	if c == nil {
 		return errors.New("nil LoggingConfig")
 	}
 
-	err := validateLogFilePath(&c.LogFilePath)
+	consoleLogger, err = NewConsoleLogger(&c.Console)
 	if err != nil {
 		return err
 	}
 
-	consoleLogger, err = NewConsoleLogger(&c.Console)
+	// If there's nowhere to specified put log files, we'll log an error, but we'll continue anyway.
+	if !hasLogFilePath(&c.LogFilePath, defaultLogFilePath) {
+		Errorf(KeyAll, "No logFilePath configured, and --defaultLogFilePath flag is not set. Unable to write log files!")
```

Comment:
"No logFilePath configured, and --defaultLogFilePath flag is not set. Log files required for product support are not being generated."